lib/mongoose_push/api/v1/h2_handler.ex

```diff
@@ -0,0 +1,12 @@
+defmodule MongoosePush.API.V1.H2Handler do
```

Comment:
This module is not specific to HTTP/2 - HTTP 1.1 is also supported here. I would prefer to name this module something like e.g. `ResponseEncoder`.